src/components/ModalManager.js

```diff
@@ -26,10 +26,16 @@ const ModalManager = (props) => {
 
```

Comment:
Check if user is already active on room
`const unregistered = !content.activeUsers[user.uid];`